<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="1.5s"
		durationms="1503"
		type="root"
		rid="cd7df611a75ae40"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="alex.kwie"
		rxversion="8.1.1.0"
		host="ALEXK-LT"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="en-US"
		screenresolution="3840x1080"
		timestamp="7/3/2018 3:21:39 PM"
		timestampiso="2018-07-03T20:21:39"
		endtime="7/3/2018 3:21:41 PM"
		timeoutfactor="1.3">
		<detail>
			Ranorex Testscript
		</detail>
		<item
			timeRelativeToTestSuiteStartTime="00:01.498"
			timeRelativeToTestModuleStartTime="00:01.499"
			timeWallClock="15:21:41"
			time="00:01.498"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
